### PR TITLE
OpenQASM Local Simulator and OpenQASM default IR

### PR DESCRIPTION
**Pull request by ajberdy**
*Issue #, if available:*

*Description of changes:*

- Support OpenQASM simulation with the LocalSimulator 🚀
- Switch default IR to OpenQASM

*Testing done:*

Unit/integ

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [x] I have read the [CONTRIBUTING](https://github.com/aws/amazon-braket-sdk-python/blob/main/CONTRIBUTING.md) doc
- [x] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/amazon-braket-sdk-python/blob/main/CONTRIBUTING.md#commit-your-change)
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-braket-sdk-python/blob/main/README.md) and [API docs](https://github.com/aws/amazon-braket-sdk-python/blob/main/CONTRIBUTING.md#documentation-guidelines) (if appropriate)

#### Tests

- [x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have checked that my tests are not configured for a specific region or account (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: examples/bell.py [hunk 1]
```diff
@@ -10,13 +10,57 @@
...
 
 from braket.aws import AwsDevice
 from braket.circuits import Circuit
 
 device = AwsDevice("arn:aws:braket:::device/quantum-simulator/amazon/sv1")
 
 # https://wikipedia.org/wiki/Bell_state
-bell = Circuit().h(0).cnot(0, 1)
+
+bell = Circuit()
```
Comment (speller26) [inline examples/bell.py:26]: This circuit, as well as the entire file, is now a misnomer

File: examples/job.py [hunk 2]
```diff
@@ -19,6 +19,7 @@
 
 
 def run_job():
+    assert 0
```
[the remark this answers is not in the document]
Comment (ajberdy) [reply examples/job.py]: Artifact -- deleted

File: examples/bell.py (hunk 1 above)
Comment (ajberdy) [reply examples/bell.py]: Yep good catch

**Review by speller26 — APPROVED** [review]

**Review by speller26 — DISMISSED** [review]

**Comment by ajberdy** [comment]
Docs didn't find the new simulator release and can't be retired. Doc string change incoming to retry tests

**Review by speller26 — APPROVED** [review]

**Review by speller26 — APPROVED** [review]

**Pull request merged**